Migrate iOS16 expectations to OpenSource (242650)

robert-jenner opened the pull request:
#### 4ddffb9ef5787e0cf9f15bb73e28c8686920c22b
<pre>
Migrate iOS16 expectations to OpenSource (242650)
<a href="https://bugs.webkit.org/show_bug.cgi?id=242650">https://bugs.webkit.org/show_bug.cgi?id=242650</a>

Reviewed by Ryan Haddad.

* LayoutTests/platform/ios-15/TestExpectations: Added.
* LayoutTests/platform/ios-wk2/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/cookies/value/value-ctl-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/dom/documents/resource-metadata-management/document-cookie-expected.txt: Added.
* LayoutTests/platform/ios/mathml/presentation/mo-stretch-expected.txt:
* LayoutTests/platform/ios/mathml/presentation/roots-expected.txt:
* LayoutTests/platform/ios/mathml/radical-fallback-expected.txt:
* LayoutTests/platform/ios/svg/W3C-SVG-1.1/masking-path-04-b-expected.txt:
* LayoutTests/platform/ios/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/ios/svg/custom/use-invalid-pattern-expected.txt: Added.
* LayoutTests/platform/ios/svg/custom/use-invalid-style-expected.txt: Added.
* LayoutTests/platform/ios/svg/hixie/error/012-expected.txt:
* Tools/Scripts/webkitpy/port/ios.py:
(IOSPort):
* Tools/Scripts/webkitpy/port/ios_device_unittest.py:
* Tools/Scripts/webkitpy/port/ios_simulator_unittest.py:
(IOSSimulatorTest.make_port):
(IOSSimulatorTest.test_layout_test_searchpath_with_apple_additions):
(IOSSimulatorTest.test_layout_test_searchpath_without_apple_additions):
(IOSSimulatorTest.test_layout_searchpath_wih_device_type):
* Tools/Scripts/webkitpy/port/ios_testcase.py:
(IOSTest.make_port):

Canonical link: <a href="https://commits.webkit.org/252930@main">https://commits.webkit.org/252930@main</a>
</pre>

ryanhaddad commented on LayoutTests/platform/ios-15/TestExpectations:
```diff
@@ -0,0 +1,12 @@
+#test to verify expectations set here actually work
+mathml/presentation/mo-stretch.html [ Failure ]
+mathml/presentation/roots.xhtml [ Failure ]
+mathml/radical-fallback.html [ Failure ]
```
Does this file need to be removed from the commit?

robert-jenner replied:
I think it’s safe to leave it. I still need to figure out the re-base situation for those tests. But after this change lands I can then work out the rebases needed for IOS15 and iOS16.

ryanhaddad replied:
Ok. In that case, the comment above these expectations should change since it isn't a test, and we probably need a bug tracking re-enabling them.

robert-jenner replied:
I have removed these expectations, and will add them back in a separate commit. Moving to land this.

robert-jenner committed:
Migrate iOS16 expectations to OpenSource (242650)
https://bugs.webkit.org/show_bug.cgi?id=242650

Reviewed by Ryan Haddad.

* LayoutTests/platform/ios-15/TestExpectations: Added.
* LayoutTests/platform/ios-wk2/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/cookies/value/value-ctl-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/dom/documents/resource-metadata-management/document-cookie-expected.txt: Added.
* LayoutTests/platform/ios/mathml/presentation/mo-stretch-expected.txt:
* LayoutTests/platform/ios/mathml/presentation/roots-expected.txt:
* LayoutTests/platform/ios/mathml/radical-fallback-expected.txt:
* LayoutTests/platform/ios/svg/W3C-SVG-1.1/masking-path-04-b-expected.txt:
* LayoutTests/platform/ios/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/ios/svg/custom/use-invalid-pattern-expected.txt: Added.
* LayoutTests/platform/ios/svg/custom/use-invalid-style-expected.txt: Added.
* LayoutTests/platform/ios/svg/hixie/error/012-expected.txt:
* Tools/Scripts/webkitpy/port/ios.py:
(IOSPort):
* Tools/Scripts/webkitpy/port/ios_device_unittest.py:
* Tools/Scripts/webkitpy/port/ios_simulator_unittest.py:
(IOSSimulatorTest.make_port):
(IOSSimulatorTest.test_layout_test_searchpath_with_apple_additions):
(IOSSimulatorTest.test_layout_test_searchpath_without_apple_additions):
(IOSSimulatorTest.test_layout_searchpath_wih_device_type):
* Tools/Scripts/webkitpy/port/ios_testcase.py:
(IOSTest.make_port):

Canonical link: https://commits.webkit.org/252930@main
